Owncloud iOs Client/Tabs/FileTab/FilesViewController.m

```diff
@@ -234,6 +234,23 @@ - (void) viewDidAppear:(BOOL)animated {
             self.navigationItem.titleView = imageView;
         }
     }
+    
+
+//    //If is a new user set the file list
+//    if (app.isNewUser) {
```

Comment:
This code can not be commented. It is used to put the tableView as it should after a change of user